pkg/controller/logstorage/dashboards/dashboards_controller.go

```diff
@@ -347,6 +357,8 @@ func (d DashboardsSubController) Reconcile(ctx context.Context, request reconcil
 		KibanaHost:                 kibanaHost,
 		KibanaScheme:               kibanaScheme,
 		KibanaPort:                 kibanaPort,
+		KibanaDomain:               kibanaDomain,
+		KibanaPortAsInt:            kibanaPortAsInt,
```

Comment:
We definitely don't need a KibanaPort and KibanaPortAsInt - they should always be the same value, and the render code can just `fmt.Sprintf("%s", cfg.KibanaPort)` where it needs it as a string.